src/Pelagos/Bundle/AppBundle/Resources/public/js/tree.js

```diff
@@ -167,11 +160,12 @@ function updateTree(tree) {
     });
 
     $("#" + tree.name).bind("loaded.jstree", function(event, data) {
+        console.log('loaded');
         if (typeof tree.onload !== 'undefined') {
```

Comment:
Works when any either one of the if tree.onload block is removed. But I am not sure that I understand the whole function

Reply:
Removed the second eval for now, this was broken in Production, so I don't know what it should do.